pyproject.toml

```diff
@@ -140,8 +140,8 @@ junit_family = "xunit2"
 filterwarnings = [
     "error",
     'ignore:"@coroutine" decorator is deprecated since Python 3\.8, use "async def" instead:DeprecationWarning',
-    # TODO: needed by AnyIO in Python 3.9, try to remove after an AnyIO upgrade
-    'ignore:The loop argument is deprecated since Python 3\.8, and scheduled for removal in Python 3\.10:DeprecationWarning',
+    # TODO: needed by asyncio in Python 3.9.7 https://bugs.python.org/issue44815, try to remove on 3.9.8
+    'ignore:The loop argument is deprecated since Python 3\.8, and scheduled for removal in Python 3\.10:DeprecationWarning:asyncio',
```

Comment:
this `:asyncio` bit means it only hides the warning if the warning is raised by something in `asyncio` and not from elsewhere, eg `fastapi`

Reply:
it's actually not pytest's fault: https://docs.python.org/3/library/warnings.html#describing-warning-filters